File: STP/src/gestionAplicacion/Vehiculos/Vehiculo.java
package gestionAplicacion.Vehiculos;
import gestionAplicacion.Personas.*;

import java.util.ArrayList;

import gestionAplicacion.Destinos.*;


public class Vehiculo {
	private Conductor conductor;
	private ArrayList<Conexion> ruta = new ArrayList<Conexion>();
	private String placa;
	private int modelo;
	private String marca;
	private double precio;
	
	//constructores
	public Vehiculo(Conductor conductor, ArrayList<Conexion> ruta, String placa, int modelo) {
		super();
		this.conductor = conductor;
		this.ruta = ruta;
		this.placa = placa;
		this.modelo = modelo;
	}
	
	public Vehiculo(String marca,double precio,int modelo,String placa) {
		this.marca = marca;
		this.precio = precio;
		this.placa = placa;
		this.modelo = modelo;
	}
	
	//getters and setters 
	public Conductor getConductor() {
		return conductor;
	}
	public void setConductor(Conductor conductor) {
		this.conductor = conductor;
	}
	public ArrayList<Conexion> getRuta() {
		return ruta;
	}
	public void setRuta(ArrayList<Conexion> ruta) {
		this.ruta = ruta;
	}
	public String getPlaca() {
		return placa;
	}
	public void setPlaca(String placa) {
		this.placa = placa;
	}
	public int getModelo() {
		return modelo;
	}
	public void setModelo(int modelo) {
		this.modelo = modelo;
	}

	public String getMarca() {
		return marca;
	}

	public void setMarca(String marca) {
		this.marca = marca;
	}

	public double getPrecio() {
		return precio;
	}

	public void setPrecio(double precio) {
		this.precio = precio;
	}
	
	
	
}

File: STP-Python/src/gestionAplicacion/Vehiculos/Vehiculo.py
class Vehiculo ():

    def __init__(self, conductor, ruta, marca, precio, placa, modelo):
        self._conductor = conductor
        self._ruta = ruta
        self._marca = marca
        self._precio = precio
        self._placa = placa
        self._modelo = modelo
    
    def getConductor(self):
        return self._conductor
    def getRuta(self):
        return self._ruta
    def getMarca(self):
        return self._marca
    def getPlaca(self):
        return self._placa
    def getModelo(self):
        return self._modelo
    def getPrecio(self):
        return self._precio

    def setConductor(self, conductor):
        self._conductor = conductor
    def setRCuta(self, ruta):
        self._ruta = ruta
    def setMarca(self, marca):
        self._marca = marca
    def setPrecio(self, precio):
        self._precio = precio
    def setPlaca(self, placa):
        self._placa = placa
    def setModelo(self, modelo):
        self._modelo = modelo
    
    def presentacion(self):
        sal = str(self._marca) + ", " + str(self._modelo)
        return sal 
